nats/src/main/java/io/micronaut/nats/ConsumerRegistry.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.nats;
+
+import java.util.Set;
+
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.core.annotation.Nullable;
+import io.nats.client.Consumer;
+import io.nats.client.Subscription;
+
+/**
+ * A registry for created Nats consumers.
+ *
+ * @author jgrimm
+ */
+public interface ConsumerRegistry {
```

Comment:
can you add `@since` with the next version this will go into

Reply:
done